Riku/Riku/Patch.h

```diff
@@ -0,0 +1,15 @@
+#pragma once
+#include <vector>
+#include "PlayerPatch.h"
+class Patch
+{
+public:
+	Patch() {};
+	Patch(PlayerPatch playerPatch) {
+		playerPatches.insert({ playerPatch.player, playerPatch });
+	}
+	std::map<int,PlayerPatch> playerPatches;
```

Comment:
z tego co mi wiadomo klucze w std::map muszą być unikatowe.
To chyba trochę psuje koncepcję, nie?

Reply:
będzie trzeba też łączyć zmiany surowców w playerPatch albo pilnować, żeby każdą zmianę aplikować przed zrobieniem następnej. Jak tylko się w tym nie pogubisz to jest git ;)

Reply:
Po przemyśleniu wolałbym nie zmieniać tej mapy na prywatną. Idea patch jest taka, że to jest prosta struktura, którą można swobodnie zmieniać przed zaaplikowaniem do stanu gry. Chyba musimy zaufać, że żadna funkcja nie zrobi z nim nic głupiego (na zewnątrz patcha nie wypuszczamy, więc nie powinno być ciężko szukać ewentualnych błędów)